Cloudinary/Classes/ios/Video/CLDVideoPlayer.swift

```diff
@@ -207,7 +207,6 @@ extension CLDVideoPlayer {
     func handleTimeControlStatusChanged(_ status: AVPlayer.TimeControlStatus) {
         switch status {
         case .playing:
-            break
```

Comment:
This will create a new bug... not both this case and the next will run.
If there is a default both this case and default will run.
 
 Move the 'break' after line 211, do not remove it.

Reply:
you do not need a `break` in swift if there are any statements in the `case`

Reply:
Yes you do

js```
function x(name) {
    switch (name){
      case 'a':
        console.log('a');
      case 'b':
       console.log('b')
       break;
      default:
       console.log('default')  
    }
}

In this function both 'a' and the default will be run.

Reply:
https://docs.swift.org/swift-book/documentation/the-swift-programming-language/controlflow/#No-Implicit-Fallthrough

Reply:
i didnt see it swift ...lol